.github/workflows/zmq-lib.yml

```diff
@@ -0,0 +1,175 @@
+      credentials:
+        username: ${{ secrets.DOCKER_REGISTRY_ID }}
+        password: ${{ secrets.DOCKER_REGISTRY_PASSWD }}
+
+    name: package ${{ matrix.distrib }}
+
+    steps:
+      - name: package deb
+        run: |
+          apt-get install -y dpkg-dev wget
+          cd /github/home
+          wget -O - https://github.com/zeromq/libzmq/releases/download/v4.3.5/zeromq-4.3.5.tar.gz|tar zxvf -
+          cd zeromq-4.3.5
+          ./configure
+          make
+          make install
+          cd /github/home
+          wget -O - https://github.com/zeromq/libzmq/archive/refs/tags/v4.3.5.tar.gz|tar zxvf -
+          cd libzmq-4.3.5
+          apt-get install -y debhelper dh-autoreconf libkrb5-dev libnorm-dev libpgm-dev libsodium-dev libunwind8-dev libnss3-dev libgnutls28-dev libbsd-dev pkg-config asciidoc xmlto
+          ln -s packaging/debian
+          dpkg-buildpackage -us -uc -nc
+        shell: bash
+
+      - name: cache deb
+        uses: actions/cache/save@13aacd865c20de90d75de3b17ebe84f7a17d57d2 # v4.0.0
+        with:
+          path: /github/home/*.deb
+          key: ${{ github.sha }}-${{ github.run_id }}-deb-${{ matrix.distrib }}
+
+  deliver-rpm:
+    runs-on: [self-hosted, common]
+    needs: [get-version, package-rpm]
+    if: ${{ contains(fromJson('["testing", "unstable"]'), needs.get-version.outputs.stability) }}
+
+    strategy:
+      matrix:
+        distrib: [el8, el9]
+
+    steps:
+      - name: Delivery
+        uses: ./.github/actions/rpm-delivery
+        with:
+          module_name: libzmq
+          distrib: ${{ matrix.distrib }}
+          version: ${{ needs.get-version.outputs.version }}
+          artifactory_token: ${{ secrets.ARTIFACTORY_ACCESS_TOKEN }}
+          cache_key: ${{ github.sha }}-${{ github.run_id }}-rpm-${{ matrix.distrib }}
+          stability: ${{ needs.get-version.outputs.stability }}
+
+  deliver-deb:
+    runs-on: [self-hosted, common]
+    needs: [get-version, package-deb]
+    if: ${{ contains(fromJson('["testing", "unstable"]'), needs.get-version.outputs.stability) }}
+
+    strategy:
+      matrix:
+        distrib: [bullseye, bookworm]
+
+    steps:
+      - name: Delivery
+        uses: ./.github/actions/deb-delivery
+        with:
+          module_name: libzmq
+          distrib: ${{ matrix.distrib }}
+          version: ${{ needs.get-version.outputs.version }}
+          artifactory_token: ${{ secrets.ARTIFACTORY_ACCESS_TOKEN }}
+          cache_key: ${{ github.sha }}-${{ github.run_id }}-deb-${{ matrix.distrib }}
+          stability: ${{ needs.get-version.outputs.stability }}
+
+  promote:
+    needs: [get-version]
+    if: ${{ contains(fromJson('["stable"]'), needs.get-version.outputs.stability) && github.event_name != 'workflow_dispatch' }}
+    runs-on: [self-hosted, common]
+    strategy:
+      matrix:
+        distrib: [el8, el9, bullseye, bookworm, jammy]
```

Comment:
@sdepassio, @paul-oureib asked if jammy package has to be promoted ? 
https://github.com/centreon/centreon/pull/3278#pullrequestreview-1864683305

Reply:
this step is a copy/paste from centreon-collect.yml

Reply:
indeed. It is managed (excluded) in the reusable action called promote-to-stable